web-admin/src/features/organizations/user-management/dialogs/CreateGuestsDialog.svelte

```diff
@@ -0,0 +1,297 @@
+            try {
+              await handleCreate(email);
+              return { index, email, success: true };
+            } catch {
+              return { index, email, success: false };
+            }
+          }),
+        );
+
+        const succeeded: string[] = [];
+        const failed: string[] = [];
+        results
+          .sort((a, b) => a.index - b.index)
+          .forEach(({ email, success }) =>
+            success ? succeeded.push(email) : failed.push(email),
+          );
+
+        if (succeeded.length > 0) {
+          eventBus.emit("notification", {
+            type: "success",
+            message: `Invited ${succeeded.length} guest${succeeded.length > 1 ? "s" : ""} as ${selectedRole}`,
+          });
+        }
+
+        if (failed.length > 0) failedInvites = failed;
+
+        // Invalidate lists
+        await queryClient.invalidateQueries({
+          queryKey:
+            getAdminServiceListOrganizationMemberUsersQueryKey(organization),
+        });
+        await queryClient.invalidateQueries({
+          queryKey:
+            getAdminServiceListOrganizationInvitesQueryKey(organization),
+        });
+
+        if (failedInvites.length === 0) {
+          open = false;
+          selectedProjects = [];
+          selectedRole = "viewer";
+        }
+      },
+      validationMethod: "oninput",
+    },
+  );
+
+  $: hasInvalidEmails = $form.emails.some(
+    (e, i) => e.length > 0 && $errors.emails?.[i] !== undefined,
+  );
+
+  function roleLabel(role: string) {
+    if (role === "admin") return "Admin";
+    if (role === "editor") return "Editor";
+    return "Viewer";
+  }
+</script>
+
+<Dialog
+  bind:open
+  onOutsideClick={(e) => {
+    e.preventDefault();
+    open = false;
+    failedInvites = [];
+    selectedProjects = [];
+    selectedRole = "viewer";
+  }}
+  onOpenChange={(open) => {
+    if (!open) {
+      failedInvites = [];
+      selectedProjects = [];
+      selectedRole = "viewer";
+    }
+  }}
+>
+  <DialogTrigger asChild>
+    <div class="hidden"></div>
+  </DialogTrigger>
+  <DialogContent class="translate-y-[-200px]">
+    <DialogHeader>
+      <DialogTitle>Create guest users</DialogTitle>
+    </DialogHeader>
+    <form
+      id={formId}
+      on:submit|preventDefault={submit}
+      class="w-full"
+      use:enhance
+    >
+      <MultiInput
+        id="emails"
+        placeholder="Add emails, separated by commas"
+        contentClassName="relative"
+        bind:values={$form.emails}
+        errors={$errors.emails}
+        singular="email"
+        plural="emails"
+      />
+
+      <!-- Project multi-select -->
+      <div class="mt-3">
+        <div class="text-xs font-medium mb-1">Project access</div>
+        {#if $projectsQuery?.isLoading}
+          <div class="text-xs text-slate-500">Loading projects…</div>
+        {:else if $projectsQuery?.error}
+          <div class="text-xs text-red-500">Failed to load projects</div>
```

Comment:
We should display the error as well. Perhaps the full page error triggers? Please test it once, you can manually throw error in the tanstack query methods.